### DISCUSSION TITLE
[Soundboard] Functionality to play sounds stored on the server?

### DISCUSSION THREAD

**Discussion by Plankt0n** [jagrosh/MusicBot#844] [Ideas]
I was wondering if it would be possible to add a function to play locally stored .mpX on the server. For instance..

A folder named customsounds where you could put short soundbits, "soundboard" sounds. And being able to play them on discord

?play sound1 

And it would play the sound.mp3 for instance located under customsounds.

**Comment by MichailiK** [comment]
You can already partially do that using a relative path to the file. `play customsound/sound.mp3`. ~~You can also use transformations, but those are experimental & undocumented right now.~~

**Reply by Plankt0n** [reply]
Alright that is quite interesting. I did not know that. So simply make a folder named ABCD in the root where the JAR is located, place a file abcd.mp3 in there, 

go to discord and do ?play ABCD/abcd.mp3?

**Reply by MichailiK** [reply]
Yes, that should work.

**Reply by Plankt0n** [reply]
Alright I tested it, it works. So I guess my re-formated request would be if it's possible to make it more "easier" instead of typing ?play soundfolder/sound.mp3 perhaps either an alias to the ?play command defined in config so;

?play ACDC would simply start finding ac/dc on youtube, play it like normal behavior whilst; 
?playc acdc would play the sound acdc.mp3 located in a subfolder? (c as in custom, could be whatever)

**Reply by MichailiK** [reply]
~~I think transformations should make something similar possible. They allow you to "rewrite" the play command, for example, you can make it so `?play s/{text}` be rewritten to `?play soundfolder/{text}.mp3`. Like I previously mentioned though, they are still experimental, so it'll have to wait for now.~~

I misunderstood how transforms work. This isn't (easily) possible with transforms. So for now, you'll have to do do `?play soundfolder/sound.mp3`.
